dart/lib/src/sentry_options.dart

```diff
@@ -297,6 +300,14 @@ class SentryOptions {
   bool isTracingEnabled() {
     return tracesSampleRate != null || tracesSampler != null;
   }
+
+  @internal
+  late SentryExceptionFactory exceptionFactory =
+      SentryExceptionFactory(this, stackTraceFactory);
```

Comment:
Since `stackTraceFactory` now can be accessed via `options.stackTraceFactory`, we don't need to pass `stackTraceFactory` anymore.